tests/framework/e2e/cluster.go

```diff
@@ -571,6 +571,9 @@ func (cfg *EtcdProcessClusterConfig) EtcdServerProcessConfig(tb testing.TB, i in
 	}
 
 	defaultValues := values(*embed.NewConfig())
+	// The default pre-vote config in previous versions may not equal to that in current version.
+	// Clear prevote's default value so cluster always add pre-vote flag even if it uses default value.
+	defaultValues["pre-vote"] = ""
```

Comment:
Could you add a check that just checks the etcd binary version and clears default just for v3.4?

Reply:
Refine the fix to apply only to versions prior to 3.5. Also add an ExplicitFlags config in case we encounter similar issues in future.